README_EN.md

```diff
@@ -3,62 +3,62 @@
 Make Minecraft Client Great Again!
 
 ## Description
-Oh My Minecraft Client is a client-side mod, which helps you manage your hands. It's written for only Fabric loader.
+Oh My Minecraft Client is a client-side mod, which helps you manage your hands. The mod is written for the Fabric Loader.
 
 ## NOTE: My Minecraft Client also requires the MaLiLib library mod by masady:
 https://www.curseforge.com/minecraft/mc-mods/malilib
 
 The default hotkey to open the in-game config GUI is O + C
 
-## Feature
+## Features
 
 ### Disable Break Scaffolding
 
-Player can only break the scaffolding with the items in **breakScaffoldingWhiteList**.
+You can break the scaffolding with the items in **breakScaffoldingWhiteList**.
 
 ### Disable Move Down In Scaffolding
 
-Player can only move down in scaffolding when the item of **moveDownInScaffoldingWhiteList** in hand
+You can move down in scaffolding when the item of **moveDownInScaffoldingWhiteList** in your hand.
 
-### Forced break cooling
+### Forced Break Cooling
 
-Players will have a 5gt cooldown after breaking the block instantaneously, which prevent you from accidentally destroying components in Haste 2 effect.
+Players will have a 5GT cooldown after breaking the block instantaneously, which will prevent you from accidentally destroying components with the Haste 2 effect.
```

Comment:
```suggestion
Players will have a 5gt cooldown after breaking the block instantaneously, which will prevent you from accidentally destroying components with the Haste 2 effect.
```

Reply:
5gt is written lowercase universally in technical minecraft, from what I have seen.